travis/deploy_pull_request.sh

```diff
@@ -31,7 +31,7 @@ function publish_pull_request_urls_in_github {
   # Check GITHUB_PR_COMMENTS if `gnosis-info` exists
   # If present, do nothing as we want to reduce noise
   # Else, comment URL
-  IS_COMMENT_PRESENT=$(curl $GITHUB_PR_COMMENTS | grep -q "\"login\":\s*\"$PREDICATE\"" && echo "true" || echo "false")
+  IS_COMMENT_PRESENT=$(curl -s -H "Authorization: token ${GITHUB_GNOSIS_INFO_API_TOKEN}" $GITHUB_PR_COMMENTS | grep -q "\"login\":\s*\"$PREDICATE\"" && echo "true" || echo "false")
```

Comment:
Not this PR, but it's fine 👍 
Can you make sure the API token is set in travis?

Reply:
It is set in travis obviously
It's used in `curl POST` to post PRaul comment, a couple lines below